cdap-ldap-role/src/main/java/io/cdap/cdap/security/authorization/ldap/role/permission/RolePermissionConverter.java

```diff
@@ -152,12 +157,15 @@ public static List<EntityTypeWithPermission> convertToEntityTypeWithPermission(R
           new EntityTypeWithPermission(EntityType.ARTIFACT, StandardPermission.LIST),
           new EntityTypeWithPermission(EntityType.ARTIFACT, StandardPermission.UPDATE),
           new EntityTypeWithPermission(EntityType.ARTIFACT, StandardPermission.DELETE),
-          new EntityTypeWithPermission(EntityType.DATASET, StandardPermission.CREATE)
+          new EntityTypeWithPermission(EntityType.DATASET, StandardPermission.CREATE),
+          new EntityTypeWithPermission(EntityType.SYSTEM_APP_ENTITY, StandardPermission.USE),
```

Comment:
Why does deploying artifacts require USE and UPDATE permissions for system app entities?

Reply:
fixed